tests/phpunit/Mutation/FileMutationGeneratorTest.php

```diff
@@ -102,6 +102,8 @@ public function test_it_generates_mutations_for_a_given_file(): void
             []
         );
 
+        $mutations = iterator_to_array($mutations, false);
```

Comment:
why is it necessary to convert it to an array before?

Reply:
It is treated like an array there. E.g. there's an `assertArrayHasKey` few lines below.